src/utils.rs

```diff
@@ -63,8 +63,16 @@ pub fn get_path_for_sym(file_name: &str, id: &str) -> PathBuf {
     let mut pb = get_base(file_name);
 
     pb.push(id);
-    pb.push(file_name);
-    pb.set_extension("sym");
+
+    if file_name
+        .rsplit_once('.')
+        .map_or(false, |(_, ext)| ext.to_lowercase() == "pdb")
+    {
+        pb.push(file_name);
+        pb.set_extension("sym");
+    } else {
+        pb.push(format!("{file_name}.sym"));
+    }
```

Comment:
To avoid the `map_or`, I'd write this as follows:

```rust
    let sym_filename = match file_name.rsplit_once('.') {
        Some((filename_without_pdb, "pdb")) => {
            format!("{filename_without_pdb}.sym")
        }
        _ => format!("{file_name}.sym"),
    };
    pb.push(sym_filename);
```

Reply:
Oh I messed up the `to_lowercase`. Then probably:

```rust
    let sym_filename = match file_name.rsplit_once('.') {
        Some((filename_without_pdb, ext)) if ext.to_lowercase() == "pdb" => {
            format!("{filename_without_pdb}.sym")
        }
        _ => format!("{file_name}.sym"),
    };
    pb.push(sym_filename);
```

Reply:
Done!